src/node/build.ts

```diff
@@ -1,11 +1,11 @@
+/* eslint-disable no-console */
```

Comment:
oops

Reply:
uhmm, that eslint hint was in the middle of the imports, I just move it to the top

Reply:
I can add it to all console.log